CHANGELOG.md

```diff
@@ -24,6 +24,11 @@
   for large files.  
   [Niil Öhlin](https://github.com/niilohlin)
 
+* Add new opt-in rule `hash_function` to warn against empty wrong hashValue
+  implementation.
```

Comment:
Nit: we ask for two trailing spaces here to cause markdown to render this as a line break. See https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes

Reply:
Thanks!